### ISSUE TITLE
Doesn't compile on M1/M2 using SPM

### ISSUE THREAD

**Issue by sidan5** [lascic/UIOnboarding#18]
Get this error: 
    UIOnboardingHelper.swift:9:8 Could not find module 'UIOnboarding' for target 'x86_64-apple-ios-simulator'; found: arm64-apple-ios-simulator, at: Build/Products/Debug-iphonesimulator/UIOnboarding.swiftmodule
![Screenshot 2023-09-01 at 10 32 41](https://github.com/lascic/UIOnboarding/assets/1595490/dfee26cc-4dbc-41e3-bf5a-0fdf22453f82)

**Comment by sidan5** [comment]
This happens also on the Demo project. So should be easy to debug (if I knew how to contribute I would).

Is there a way to integrate using Cocoapods (same issues happens from time to time with other frameworks as Firebase)?

**Comment by lascic** [comment]
Thanks for reporting. Is this in the newest Xcode 15 beta?

**Comment by sidan5** [comment]
Nope. Xcode stable release…On 1 Sep 2023, at 15:04, Lukman Aščić ***@***.***> wrote:﻿
Thanks for reporting. Is this in the newest Xcode 15 beta?

**Comment by sidan5** [comment]
> Thanks for reporting. Is this in the newest Xcode 15 beta?

Nope. Latest formal release...

**Comment by lascic** [comment]
Checking in on how the current situation is with the error. Is it still appearing?
